Marlin/ultralcd.cpp

```diff
@@ -84,7 +84,7 @@ uint8_t lcd_status_update_delay = 1, // First update one loop delayed
         lcd_status_message_level;    // Higher level blocks lower level
 
 #if ENABLED(STATUS_MESSAGE_SCROLLING)
-  #define MAX_MESSAGE_LENGTH max(2 * CHARSIZE * LCD_WIDTH, LONG_FILENAME_LENGTH)
+  #define MAX_MESSAGE_LENGTH max(CHARSIZE * LCD_WIDTH, LONG_FILENAME_LENGTH)
```

Comment:
The reason that "2 *" is here is because you can set a longer message and it will scroll.

Reply:
ok,  I though scroll was for "long" filenames